.github/workflows/integration-tests.yml

```diff
@@ -56,15 +56,32 @@ jobs:
             # See:
             #   https://github.com/nh2/static-haskell-nix/pull/27#issuecomment-502652181
             #   https://github.com/NixOS/nix/issues/2733
-            (for i in {1..5}; do bash <(curl -sSL https://nixos.org/nix/install) && exit 0; done; exit 1)
+            (for i in {1..5}; do bash <(curl -sSL https://nixos.org/nix/install) --no-daemon && exit 0; done; exit 1)
             . ~/.nix-profile/etc/profile.d/nix.sh
-            nix-channel --add https://nixos.org/channels/nixos-19.09 nixpkgs
+            nix-channel --add https://nixos.org/channels/nixos-22.05 nixpkgs
             nix-channel --update # Get GHC 8.2.2
           elif [[ "${{ matrix.os }}" == "windows-latest" ]]
           then
             choco install nsis-unicode -y
           fi
 
+          if [[ "${{ matrix.release-args }}" == "--alpine" ]]
+          then
+              touch ~/.stack/config.yaml
+              cat > ~/.stack/config.yaml <<EOF
+              extra-include-dirs:
+              - /usr/include
+              extra-lib-dirs:
+              - /lib
+              - /usr/lib
+          EOF
+          fi
+
+          if [[ "${{ matrix.os }}" == "windows-latest" ]]
+          then
+              stack exec pacman -- --sync --refresh --noconfirm autoconf
+          fi
```

Comment:
I was interested in this addition. May I ask, what was the problem with the `windows-latest` CI that required the MSYS2 `autoconf` package to be downloaded?

Reply:
IIRC, it was required when I was using `stack-macos.yaml` for Windows. I have removed it now, Thanks!